demo/src/main/java/com/orange/ods/demo/ui/components/utilities/ComponentCustomizationBottomSheet.kt

```diff
@@ -46,6 +48,7 @@ import kotlinx.coroutines.launch
 @Composable
 fun ComponentCustomizationBottomSheetScaffold(
     bottomSheetScaffoldState: BottomSheetScaffoldState,
+    snackbarHost: @Composable (SnackbarHostState) -> Unit = { SnackbarHost(it) },
```

Comment:
`SnackbarHost(it)` can be replaced by `SnackbarHost(hostState = it)` for clarity.